task-definition.web.json

```diff
@@ -101,6 +101,14 @@
                 {
                     "name": "CLICKHOUSE_DENORMALIZED_PROPERTIES",
                     "value": "issampledevent,currentscreen,objectname"
+                },
+                {
+                    "name": "PLUGIN_SERVER_INGESTION",
+                    "value": "false"
+                },
+                {
+                    "name": "PLUGINS_CLOUD_WHITELISTED_ORG_IDS",
+                    "value": "4dc8564d-bd82-1065-2f40-97f7c50f67cf,2f3a6d88-4789-027b-6205-7b9a0e12217d"
```

Comment:
This var already is set 20 lines above, except there the second org is `57543eee-fe77-11ea-b810-0902baec2d39` and here it is `2f3a6d88-4789-027b-6205-7b9a0e12217d`, so kind of a discrepancy

Reply:
Dang, will fix.

Reply:
I'm not sure what that other organization was, but it didn't exist anymore.